src/scss/components/switch/mixins/_hxSwitchControl.scss

```diff
@@ -56,26 +67,32 @@
 
     &::-ms-check {
       border-color: $red-status-500;
+      border-color: var(--hxSwitchControl-error-focus-border-color, $red-status-500);
     }
   } @else if $pseudo-state == hover {
     &::-ms-check {
       background-color: $red-100;
+      background-color: var(--hxSwitchControl-error-hover-bgcolor, $red-100);
```

Comment:
Please use updated colors to replace `$red-XXX` -> `$red-status-XXX`.

Reply:
Changes have been done, please review.